src/xmldoc.cpp

```diff
@@ -32,6 +32,8 @@ limitations under the License.
 
 namespace libcellml {
 
+static std::regex newLineRegex("\\n");
```

Comment:
Why not simply have it in `structuredErrorCallback()`, at least until we need it elsewhere too.

Reply:
[172e825](https://github.com/cellml/libcellml/pull/1112/commits/172e82586d21b14c53d16548d3b684e28336114d)